manifests/overlays/additional/kustomization.yaml

```diff
@@ -1,13 +0,0 @@
----
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
-
-resources:
-  - ../../base
-  - code-server-notebook-imagestream.yaml
-  - rstudio-notebook-imagestream.yaml
-  - rstudio-gpu-notebook-imagestream.yaml
```

Comment:
I would suggest, keep this file
and remove all these parts.

reasoning: in this way, anyone can use additional overlay for deployment and in future for other new inclusion
people would know where to add new yaml, instead of creating the whole kustomize.